hotness/hotness_consumer.py

```diff
@@ -601,16 +606,14 @@ def _comment_on_bugzilla_with_template(
         bz_id = -1
         latest_upstream = package.version
 
-        upstream_versions = latest_upstream
-        if retrieved_versions:
-            upstream_versions = ", ".join(retrieved_versions)
-
         # Prepare dist-git URL
         dist_git_url = self.dist_git_url + "/rpms/" + package.name
         # Prepare message for bugzilla
         description = self.description_template % dict(
             latest_upstream=package.version,
-            retrieved_versions=upstream_versions,
+            retrieved_versions=", ".join(retrieved_versions)
```

Comment:
Not easily readable, it will be better to fill this variable before calling the `dict` function.

Reply:
The same was available on the line 604 you deleted :-)